out/install/Release/share/dolphin-emu/license.txt

```diff
@@ -0,0 +1,339 @@
+		    GNU GENERAL PUBLIC LICENSE
```

Comment:
Can you please clean up the commit history and don't modify the .mo files or add exe files.